src/puya/awst_build/module.py

```diff
@@ -576,6 +579,34 @@ def _process_contract_class_options(
                             )
                         else:
                             scratch_slot_reservations |= slots
+                case "state_totals":
+                    match kw_expr:
+                        case mypy.nodes.StrExpr(value="auto"):
+                            pass
+                        case mypy.nodes.CallExpr(arg_names=arg_names, args=args):
+                            arg_map = get_arg_mapping(
+                                positional_arg_names=[
```

Comment:
maybe mark these as kw-only in the stubs and then we can omit this list here? probably wouldn't want users relying on the stubs order of the attributes either way

Reply:
Removed the positional args, as the stubs were already kw-only